app/assess/models/application.py

```diff
@@ -49,3 +49,11 @@ def get_question(self, index: int):
         if self.questions:
             return self.questions[index]
         return None
+
+    def get_question_field(self, page_title):
+        fields = []
+        for question in self.questions:
+            if page_title == question.title:
+                for field in question.fields:
```

Comment:
you could just `return question.fields` here rather than creating a new list

Reply:
Yeah, I am aware of that, it's just because  "About your organisation"  pages require to loop through three pages then return otherwise question.fields would do the job for individual pages.